### PR TITLE
Initial Rust Backend

### PR DESCRIPTION
**Pull request by kesquerra**
Backend API skeleton
- Logging
- Basic API routes
- Test users
- Postgres Db Persistence
- .env configuration

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Build Backend:
```
Comment (TaylorKNoah) [inline backend/README.md:1]: Can you add a section describing how to run the backend? ie what commands  to run.

**Review by TaylorKNoah — CHANGES_REQUESTED** [review]
Can you add a section in the README describing how to run the backend? ie what commands to run.

**Comment by kesquerra** [comment]
The instructions to run the backend are in the README. As of right now, it just takes "cargo run" with rust installed.

**Comment by kesquerra** [comment]
I did update the configuration to use a .env file recently. I'll update the README to reflect that, good catch!

**Comment by kesquerra** [comment]
.env instructions added with link to example.env

**Review by adam-taitano — APPROVED** [review]
Compiled on my end, and seems to work, at least on the main '/ route.
I'll try to add some automated testing by the end of this weekend. Nice work.

**Review by TaylorKNoah — APPROVED** [review]
Looks good! Thanks!

**Comment by adam-taitano** [comment]
Sweet solid stuff!

**Pull request merged**